frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/controlData/ControlData.js

```diff
@@ -147,7 +148,32 @@ export const getControlData = (warning, onControlSelect, awsPrivateFeatureGate =
                         'install-config': { template: installConfig, encode: true, newTab: true },
                     },
                 },
-                section: 'Infrastructure providers',
+                section: 'Providers',
+            },
+            {
+                id: 'RHV',
+                logo: <RedHatLogo />,
+                title: 'cluster.create.rhv.subtitle',
+                change: {
+                    insertControlData: getControlDataRHV(),
+                    replacements: {
+                        'install-config': { template: installConfig, encode: true, newTab: true },
+                    },
+                },
+                section: 'Providers',
+            },
+            {
+                id: 'AI',
```

Comment:
This is irrelevant change which broke the Assisted Installer. Fix: https://github.com/stolostron/console/pull/1136